MREUnityRuntime/MREUnityRuntimeLib/Core/Actor.cs

```diff
@@ -875,6 +875,7 @@ private RigidBody AddRigidBody()
 			if (_rigidbody == null)
 			{
 				_rigidbody = gameObject.AddComponent<Rigidbody>();
+				//_rigidbody.maxDepenetrationVelocity = 3.0F; // set very low de-penetration velocities
```

Comment:
>//_rigidbody.maxDepenetrationVelocity = 3.0F; // set very low de-penetration velocities [](start = 4, length = 87)

should we uncomment this?

Reply:
For the moment I would leave it as a comment because depenetration helps when one remote body pushes one owned body or vice versa and even without this limit we have quite some good behavior.